app/services/create_cluster_stats_service.rb

```diff
@@ -36,6 +38,25 @@ def call
 
   private
 
+  def epochs_annual_roi
+    if last_epochs.count > 0
+      epoch_duration = (last_epochs.first.created_at - last_epochs.second.created_at)
+      return 0 unless epoch_duration && epoch_duration > 0
```

Comment:
It should be possible to shorten the condition to `epoch_duration.to_f > 0` or  `epoch_duration.to_i > 0`